source/gameengine/Ketsji/KX_Scene.cpp

```diff
@@ -2533,4 +2534,33 @@ KX_PYMETHODDEF_DOC(KX_Scene, get, "")
 	return def;
 }
 
+/* Find gameobjects matching a name, a property... */
+KX_PYMETHODDEF_DOC(KX_Scene, getMatching, "scene.getMatching(objectsName, objectsProperty...)")
+{
+	PyObject *objectslist = PyList_New(0);
+	char *objectsName = "";
+	char *objectsProperty = "";
+
+	static const char *kwlist[] = { "name", "property", NULL };
+
+	if (!PyArg_ParseTupleAndKeywords(args, kwds, "|ss:getMatching", (char **)(kwlist), &objectsName, &objectsProperty))
+		return NULL;
+
+	for (CListValue::iterator<KX_GameObject> it = m_objectlist->GetBegin(), end = m_objectlist->GetEnd(); it != end; ++it) {
+		KX_GameObject *gameobj = *it;
+		if (strlen(objectsName)) {
```

Comment:
The user want to have a list of object with giving name AND property. or a list of object with the giving name OR property ?